backend/app/controllers/spree/admin/refunds_controller.rb

```diff
@@ -8,6 +8,22 @@ class RefundsController < ResourceController
 
       rescue_from Spree::Core::GatewayError, with: :spree_core_gateway_error
 
+      def create
```

Comment:
Instead of defining our own `create` could we take advantage of `Spree::Backend::Callbacks` and do:
```ruby
create.after :perform_refund
...
def perform_refund
  @refund.perform!
end
```
?